terraform/modules/iam_user/health_check/outputs.tf

```diff
@@ -3,17 +3,17 @@ output "username" {
 }
 
 output "access_key_id_prev" {
-  value = ""
+  value = aws_iam_access_key.iam_access_key_v1.id
 }
 
 output "secret_access_key_prev" {
-  value = ""
+  value = aws_iam_access_key.iam_access_key_v1.secret
 }
 
 output "access_key_id_curr" {
-  value = aws_iam_access_key.iam_access_key_v1.id
+  value = aws_iam_access_key.iam_access_key_v12.id
```

Comment:
Is this meant to be `v2`?

Reply:
yes, thanks for catching that! I just fixed it